frontend-react/src/components/alerts/AdminFetchAlert.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
I feel like it's not really worth it to test this out since it's using StaticAlert logic verbatim 🤷